app/controllers/admins/rewards_controller.rb

```diff
@@ -39,6 +39,16 @@ def update
       render :edit, notice: e.message
     end
 
+    def import; end
+
+    def import_create
```

Comment:
why not simply `import`?

Reply:
When I have two import in routes (for get and for post methods) I have a problem with the redirect loop. Could you suggest another solution if this isn't good? 🙂

Reply:
CAn you elaborate the redirect loop issue? I would focus on solving that, and keeping one action for both actions

Reply:
it looks weird, same path works perfectly fine - I think this like is causing issues:
get 'import', to: "rewards#import_form"
it should be simply:
get 'import' - because it is inside a resources, and collection, and rewards#import_form might by ambiguous - rails routes should display all the routes with paths - check for duplicates